showcase/pom.xml

```diff
@@ -103,7 +112,7 @@
   </dependencies>
 
   <build>
-    <finalName>${project.artifactId}</finalName>
+    <finalName>showcase</finalName>
```

Comment:
Good idea.

Reply:
To be honest I did this only to be able to use Adam Biens `wad` script :sweat_smile: But thanks ;)